=== RUN   TestPanicWithBrokenPipe
=== RUN   TestPanicWithBrokenPipe/broken_pipe
=== RUN   TestPanicWithBrokenPipe/connection_reset_by_peer
--- PASS: TestPanicWithBrokenPipe (0.00s)
    --- PASS: TestPanicWithBrokenPipe/broken_pipe (0.00s)
    --- PASS: TestPanicWithBrokenPipe/connection_reset_by_peer (0.00s)
PASS
ok  	github.com/gin-gonic/gin	0.008s
FileName:  /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/recovery_test.go
TestName:  TestPanicWithBrokenPipe
Import added at line: 7
Header added at line: 131
Header added at file: /home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/recovery_test.go
go version go1.24.1 linux/amd64 ADVOCATE
=== RUN   TestPanicWithBrokenPipe
=== RUN   TestPanicWithBrokenPipe/connection_reset_by_peer
SetExitCode: other
panic: : : connection reset by peer

goroutine 22 [running]:
runtime.SetExitCodeFromPanicMsg({0x122b6e0, 0xc00017ea50})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/advocate_exit.go:82 +0x359
runtime.ExitReplayPanic({0x122b6e0, 0xc00017ea50})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/advocate_replay.go:937 +0x3c
panic({0x122b6e0, 0xc00017ea50})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/panic.go:744 +0x2f
github.com/gin-gonic/gin.TestPanicWithBrokenPipe.func1.1(0xc00017cea0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/recovery_test.go:157 +0x15b
github.com/gin-gonic/gin.(*Context).Next(0xc00017cea0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/context.go:192 +0xab
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1(0xc00017cea0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/recovery.go:94 +0xfd
github.com/gin-gonic/gin.(*Context).Next(0xc00017cea0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/context.go:192 +0xab
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest(0xc00015b6c0, 0xc00017cea0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/gin.go:689 +0x42a
github.com/gin-gonic/gin.(*Engine).ServeHTTP(0xc00015b6c0, {0x13a4220, 0xc0001f1200}, 0xc0001dcdc0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/gin.go:643 +0xbc
github.com/gin-gonic/gin.PerformRequest({0x139d7e0, 0xc00015b6c0}, {0x12dc154, 0x3}, {0x12e911d, 0x9}, {0x0, 0x0, 0x0})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/routes_test.go:31 +0x1bc
github.com/gin-gonic/gin.TestPanicWithBrokenPipe.func1(0xc0002523c0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/recovery_test.go:160 +0x255
testing.tRunner(0xc0002523c0, 0xc00012ddc0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1792 +0x1da
created by testing.(*T).Run in goroutine 20
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1851 +0x8fc

goroutine 1 [chan receive]:
testing.(*T).Run(0xc0002521e0, {0x12f8275, 0x17}, 0x1330438)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1859 +0x92c
testing.runTests.func1(0xc000252000)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:2279 +0xad
testing.tRunner(0xc000252000, 0xc000059918)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1792 +0x1da
testing.runTests(0xc000120978, {0x19434e0, 0x16f, 0x16f}, {0xc20d46ba9eada2d5, 0x8bb2fe56e5, 0x194b920})
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:2277 +0x5ea
testing.(*M).Run(0xc0001642c0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:2142 +0xc79
main.main()
	_testmain.go:815 +0xa5

goroutine 20 [chan receive]:
testing.(*T).Run(0xc0002523c0, {0x12f9a08, 0x18}, 0xc00012ddc0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1859 +0x92c
github.com/gin-gonic/gin.TestPanicWithBrokenPipe(0xc0002521e0)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/Examples/Examples_Projects/gin-master/recovery_test.go:145 +0x365
testing.tRunner(0xc0002521e0, 0x1330438)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1792 +0x1da
created by testing.(*T).Run in goroutine 1
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/testing/testing.go:1851 +0x8fc

goroutine 21 [sleep]:
time.Sleep(0xdf8475800)
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/runtime/time.go:338 +0x13c
advocate.InitTracing.func1()
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/advocate/advocate_tracing.go:72 +0x25
created by advocate.InitTracing in goroutine 20
	/home/ilian/Projects/Go_Fuzzing_Concurrency/ADVOCATE/go-patch/src/advocate/advocate_tracing.go:71 +0xd2


Exit Replay with code  3 The program panicked unexpectedly
Exit Message: panic: : : connection reset by peer

FAIL	github.com/gin-gonic/gin	0.110s
FAIL
Bugreport info: 0_1,fail
